src/multiError.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Cognite AS
+
+import { AxiosResponse } from 'axios';
+import { CogniteError } from './error';
+
+/** @hidden */
+export interface MultiErrorRawSummary<RequestType, ResponseType> {
+  succeded: RequestType[][];
+  responses: AxiosResponse<ResponseType>[];
+  failed: RequestType[][];
+  errors: (Error | CogniteError)[];
+}
+
+export class CogniteMultiError<RequestType, ResponseType> extends Error {
```

Comment:
Don't want to extend from `CogniteError`?

Reply:
`CogniteMultiError` doesn't necessarily has `status` prop
That's basically only one reason